api/v1beta2/ibmvpccluster_types.go

```diff
@@ -108,6 +108,20 @@ type VPCLoadBalancerSpec struct {
 	Subnets []VPCResource `json:"subnets,omitempty"`
 }
 
+// AdditionalListenerEnabled defines what category of VMs are added to the listener pool.
+type AdditionalListenerEnabled int
+
+const (
+	// AdditionalListenerAll add every VM to the listener pool.
+	AdditionalListenerAll AdditionalListenerEnabled = iota
+
+	// AdditionalListenerBootstrapOnly add only the bootstrap VM to the listener pool.
+	AdditionalListenerBootstrapOnly
```

Comment:
there is nothing like bootstrap term in the capi project, we need to find some generic way to deal with this problem.